references/oidc-mock/apiproxy/policies/AM-SetRedirectLocationToClientApp.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2020 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AM-SetRedirectLocationToClientApp">
+    <DisplayName>AM-SetRedirectLocationToClientApp</DisplayName>
+    <Properties/>
+    <Set>
+        <Headers>
+            <Header name="Location">{oidc.flow.token.redirect_uri}?code={flow.oidc.azcode}&amp;state={oidc.flow.token.state}</Header>
+        </Headers>
+        <StatusCode>302</StatusCode>
+        <ReasonPhrase>Found Redirect</ReasonPhrase>
+    </Set>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
```

Comment:
what do you want to happen if `oidc.flow.*` is null? do you want to change this to false?